tests/lib/rules/arrow-body-style.js

```diff
@@ -72,6 +84,7 @@ ruleTester.run("arrow-body-style", rule, {
         },
         {
             code: "var foo = () => { return bar(); };",
+            output: "var foo = () =>   bar() ;",
```

Comment:
Can we strip the extra spaces before `bar()`? It seems like you'd expect just one space because that's what is between `return` and `bar()`.